lib/engine/config/game/g_18_co.rb

```diff
@@ -0,0 +1,1260 @@
+			"110p",
+			"120p",
+			"135p",
+			"150",
+			"170",
+			"195",
+			"220",
+			"245",
+			"275",
+			"305",
+			"335",
+			"370"
+		],
+		[
+			"65",
+			"70",
+			"75p",
+			"80p",
+			"90p",
+			"100",
+			"115",
+			"130",
+			"150",
+			"170",
+			"195",
+			"215",
+			"240",
+			"265",
+			"290",
+			"320"
+		],
+		[
+			"50",
+			"55",
+			"60p",
+			"65",
+			"75",
+			"85",
+			"100",
+			"115",
+			"130",
+			"150",
+			"170",
+			"185",
+			"205"
+		],
+		[
+			"40",
+			"45",
+			"50p",
+			"55",
+			"65",
+			"75",
+			"85",
+			"100",
+			"115",
+			"130"
+		],
+		[
+			"30",
+			"35",
+			"40p",
+			"45",
+			"55",
+			"65",
+			"75",
+			"85"
+		],
+		[
+			"25",
+			"30",
+			"35",
+			"40",
+			"45",
+			"55",
+			"65"
+		],
+		[
+			"20",
+			"25",
+			"30",
+			"35",
+			"40",
+			"45"
+		],
+		[
+			"15",
+			"20",
+			"25",
+			"30",
+			"35",
+			"40"
+		],
+		[
+			"10b",
+			"15",
+			"20",
+			"25",
+			"30"
+		],
+		[
+			"10b",
+			"10b",
+			"15",
+			"20",
+			"20"
+		],
+		[
+			"10b",
+			"10b",
+			"10b",
+			"15b",
+			"15b"
+		]
+	],
+	"companies": [
+		{
+			"sym": "IMC",
+			"name": "Idarado Mining Company",
+			"value": 30,
+			"revenue": 5,
+			"desc": "Money gained from mine tokens is doubled for the owning Corporation. If owned by a Corporation, closes on purchase of “6” train, otherwise closes on purchase of “5” train.",
+			"abilities": [
+
+			]
+		},
+		{
+			"sym": "GJGR",
+			"name": "Grand Junction and Grand River Valley Railway",
+			"value": 40,
+			"revenue": 10,
+			"desc": "An owning Corporation may upgrade a yellow town to a green city in additional to its normal tile lay. Action closes the company or closes on purchase of “5” train.",
+			"abilities": [
+				{
+					"type": "tile_lay",
+					"free": true,
+					"owner_type": "corporation",
+					"when": "track",
+					"count": 1,
+					"tiles": [
+						"14",
+						"15"
+					],
+					"hexes": [
+						"B10",
+						"B22",
+						"C7",
+						"C9",
+						"C17",
+						"C21",
+						"D4",
+						"D14",
+						"D24",
+						"E5",
+						"E7",
+						"E11",
+						"F8",
+						"F12",
+						"F20",
+						"F24",
+						"H8",
+						"H12",
+						"H14",
+						"I21",
+						"I23",
+						"J6",
+						"K13"
+					]
+				}
+			]
+		},
+		{
+			"sym": "DNP",
+			"name": "Denver, Northwestern and Pacific Railroad",
+			"value": 50,
+			"revenue": 10,
+			"desc": "An owning Corporation may return a station token to its charter to gain the token cost. Corporation must always have at least one token on the board. Action closes the company or closes on purchase of “5” train.",
+			"abilities": [
+
+			]
+		},
+		{
+			"sym": "Toll",
+			"name": "Saguache & San Juan Toll Road Company",
+			"value": 60,
+			"revenue": 10,
+			"desc": "An owning Corporation receives a $20 discount on the cost of tile lays. Closes on purchase of “5” train.",
+			"abilities": [
+				{
+					"type": "tile_discount",
+					"discount": 20,
+					"terrain": "mountain"
```

Comment:
Is it only mountain costs? If so maybe the description needs to be updated.

Reply:
I see your point. I suppose I wrote it that way because the only kind of tile cost in the game is mountainous terrain, but it should be removed.

Reply:
@perwestling Terrain type is a requirement of this ability, and mountain is the only type of terrain in 18CO.